x-pack/plugins/endpoint/public/embeddables/resolver/store/data/reducer.ts

```diff
@@ -5,13 +5,22 @@
  */
 
 import { Reducer } from 'redux';
-import { DataState, ResolverAction } from '../../types';
+import {
+  DataState,
+  ResolverAction,
+  resultsEnrichedWithRelatedEventInfo,
```

Comment:
🎗️ importing so we keep the key in sync in the different places we use it. Per earlier review, it was changed from a `Symbol` to a string.